pyalgostrategypool/rsi_macd_crossover_v2.py

```diff
@@ -0,0 +1,80 @@
+import talib
```

Comment:
remove all imports, run this strategy from the python build website, clear all issues using the execution, and paste the changed code here